Header header
float64 position
int64 speed